view/src/storage/schema.sql

```diff
@@ -0,0 +1,119 @@
+-- The hash of this schema file
```

Comment:
Does this need to live inline with the Rust code? Could we keep it in `view/schema/view.sql` (rather than "migrations")?